packages/valory/services/trader_omen_gnosis/service.yaml

```diff
@@ -0,0 +1,131 @@
+name: trader_omen_gnosis
+author: valory
+version: 0.1.0
+description: A set of agents placing bets on omen
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: bafybeibg7bdqpioh4lmvknw3ygnllfku32oca4eq5pqtvdrdsgw6buko7e
+fingerprint_ignore_patterns: []
+agent: valory/trader:0.1.0:bafybeiexmyrslgxqnygcbir5ws26ltcjv5qo7e7plqapoo3j22xuqpfy64
+number_of_agents: 1
+deployment:
+  chain:
+    nft: bafybeig64atqaladigoc3ds4arltdu63wkdrk3gesjfvnfdmz35amv7faq
+    agent_id: 14
+    threshold: 1
+    cost_of_bond: 10000000000000000
+    funds:
+      agent: 0.1
+      safe: 0.5
+  local:
+    volumes:
+      data: /data
```

Comment:
Right now the deployment parameter does not have constrains so we can customise it until we find a right schema and back port the changes on the open-autonomy framework

Reply:
`deployment` name is confusing here. I would choose something different lik `on-chain-deployment`. LMK what you think.

Reply:
really what this is is the "minting" specification. I think this deserves its own config file. Will explain in person.